sms-checker/message.py

```diff
@@ -0,0 +1,70 @@
+'''This script will message the subscribers if they require notification'''
+import logging
+from os import environ as ENV
+from dotenv import load_dotenv
+from boto3 import client
+
+def send_sms(subscribers: list[dict]) -> None:
+    '''Uses boto3 to send messages to subscribers by sms'''
+    sms = get_client('sns')
+    logging.info('Client established for SNS')
+    for sub in subscribers:
+        if sub['body']:
+            message = 'STARWATCH ALERT'
+            message += f'\n\n Hello {sub['user'].capitalize()},\n\n'
+            if not sub['at']:
+                message += f"{sub['body'].capitalize()} will be visible tonight in your county!"
+            else:
+                message += f"\n{sub['body'].capitalize()} will be visible tonight"
```

Comment:
Could maybe change this to 'might' not 'will', don't want to give false promises.